src/app/features/tech-record/components/tech-record-title/tech-record-title.component.ts

```diff
@@ -74,4 +78,13 @@ export class TechRecordTitleComponent implements OnInit {
           : this.router.navigateByUrl(`/tech-records/${this.vehicleTechRecord?.systemNumber}/change-vta-visibility`)
       );
   }
+  navigateToChangeVehicleType() {
+    this.currentTechRecord$
+      .pipe(take(1))
+      .subscribe(techRecord =>
+        techRecord?.statusCode === StatusCodes.PROVISIONAL
+          ? this.router.navigateByUrl(`/tech-records/${this.vehicleTechRecord?.systemNumber}/provisional/changeVehicleType`)
+          : this.router.navigateByUrl(`/tech-records/${this.vehicleTechRecord?.systemNumber}/changeVehicleType`)
+      );
+  }
```

Comment:
Can be simplified as such:
```suggestion
this.router.navigate(['change-vehicle-type'], { relativeTo: this.route });
```

Reply:
SO many commits, so many merges, so confused. I used your(?) navigateTo() function?

Reply:
I know. That's great, cheers 🙂